daily-challenges/2024-09/16/hunghoang3110.spec.ts

```
@@ -0,0 +1,72 @@
+        bio: string;
+        rate: string;
+        color: string;
+    }
+    const currentDate = new Date();
+    const formattedDate = currentDate.toLocaleDateString().split("/");
+    const TEST_USER: TestUser = {
+        userName: 'carter.nguyen',
+        email: 'carter.nguyen@gmail.com',
+        interests: ["Technology", "Science", "Music", "Sports"],
+        country: 'canada',
+        dob: formattedDate[2] + '-' + formattedDate[1] + '-' + formattedDate[0],
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)